app/src/main/res/values-fr/strings.xml

```diff
@@ -251,6 +251,7 @@
     <string name="errorAlreadyLinkedMailbox">Cette adresse mail est déjà liée à votre compte</string>
     <string name="errorAtLeastOneRecipient">Il faut au minimum un destinataire</string>
     <string name="errorAttachmentNotFound">Pièce jointe introuvable</string>
+    <string name="errorCalendarChoiceCouldNotBeSent">Une erreur s’est produite lors de l’envoi de votre choix</string>
```

Comment:
Isn't it weird to talk about "your choice" to the user, "your reply" seems more natural to me